api-catalog-ui/frontend/.env

```diff
@@ -8,4 +8,4 @@ REACT_APP_STATUS_UPDATE_DEBOUNCE=300
 REACT_APP_CA_ENV=false
 REACT_APP_STATUS_UPDATE_SCALING_DURATION=1000
 REACT_APP_GATEWAY_URL=
-REACT_APP_ZOWE_BUILD_INFO=
+REACT_APP_ZOWE_BUILD_INFO=3.0.21-SNAPSHOT
```

Comment:
Please remove this file. It is updated automatically and there is no need to commit it every time.

Reply:
Removed